docs/src/routes/drawer/index.js

```diff
@@ -83,7 +85,32 @@ export default class DrawerPage extends Component {
         </div>
 
         <div className="mdc-typography--display1">Demo </div>
-        <div className="mdc-typography--title">not available </div>
+        <Button
```

Comment:
can you try to contain the drawer into a smaller container div?

Reply:
What should be the effect of this change?
Now the result is like on this example: http://www.material-ui.com/#/components/drawer
and I followed this example https://material.preactjs.com/component/dialog

Reply:
i am not able to find a live example, but consider something like a drawer opening/closing inside a div viewport

Reply:
I added drawer toggling inside div